plugins/generator-1.21.4/neoforge-1.21.4/templates/gui/gui_window.java.ftl

```diff
@@ -84,10 +83,8 @@ public class ${name}Screen extends AbstractContainerScreen<${name}Menu> implemen
 			</#list>
 		}
 		</#if>
-
-		<#-- updateMenuState is not implemented for checkboxes, as there is no procedure block to set checkbox state currently -->
-
 		menuStateUpdateActive = false;
+		<#-- updateMenuState is not implemented for checkboxes, as there is no procedure block to set checkbox state currently -->

```

Comment:
Unrelated changes

Reply:
@151chit best to just checkout the file from master instead of manually tinkering changes until it works :)